azurerm/internal/services/mssql/helper/sql_extended_auditing.go

```diff
@@ -40,6 +40,12 @@ func ExtendedAuditingSchema() *schema.Schema {
 					Optional:     true,
 					ValidateFunc: validation.IntBetween(0, 3285),
 				},
+
+				"monitor_enabled": {
```

Comment:
should we be more specific about what monitor is being enabled?
```suggestion
				"log_analytics_monitoring_enabled": {
```

Reply:
Hi kt, `monitor` could be `log_analytics` or `eventhub`, shall we keep the previous?

Reply:
I don't want to interfere to much, but maybe the API variable name can be helping us here:`isAzureMonitorTargetEnabled` -> `monitor_target_enabled` or `azure_monitor_target_enabled`? It's indeed the case that both can subscribe basically to the Azure Monitor diagnostic events by enabling the flag here. Technically this flag is only to show in the Azure Portal that subscribers are enabled (AFAIK), the actual configuration is done using an `azurerm_monitor_diagnostic_setting`.. 🤓😅